ansible_collections/arista/avd/plugins/plugin_utils/eos_designs_shared_utils/wan.py

```diff
@@ -131,9 +135,43 @@ def wan_local_carriers(self: SharedUtils) -> list:
 
         return list(local_carriers_dict.values())
 
+    def path_group_preference_to_eos_priority(self, path_group_preference: int | str, context_path: str) -> int | None:
+        """
+        Convert "preferred" to 1 and "alternate" to 2 and "excluded" to None. Everything else is returned as is.
+
+        Arguments:
+        ----------
+        path_group_preference (str|int): The value of the preference key to be converted. It must be either "preferred", "alternate" or an integer.
+        context_path (str): Input path context for the error message.
+        """
+        if path_group_preference == "preferred":
+            return 1
+        if path_group_preference == "alternate":
+            return 2
+        if path_group_preference == "excluded":
```

Comment:
So here we are going a bit back and forth on this with @ayushmittal-arista 

basically adding "excluded" here feels more natural but it also brings the issue that the funciton is used to verify both
`wan_path_groups.default_preference` AND `wan_virtual_topologies.policies[].path_groups[].preference`.
The first one allows for `excluded` but not the second one. And with this implementation it allows `excluded` for the second one.

@ClausHolbechArista for discussion

Reply:
from @ClausHolbechArista 

> Pathgroup LTE - it is specifically set in the VIDEO policy but because the default preference is excluded it is now showing up.
> so for excluded I think we need to only respect it for auto generated policies, and raise on other policies